doctr/models/backbones/mobilenet/tensorflow.py

```diff
@@ -150,14 +151,16 @@ def __init__(
             *conv_sequence(16, strides=2, activation=hard_swish, kernel_size=3, input_shape=input_shape)
         ]
 
-        for out, k, exp, s, use_sq, use_sw in zip(out_chans, kernels, exp_chans, strides, use_squeeze, use_swish):
+        for i, (out, k, exp, s, use_sq, use_sw) in enumerate(
+            zip(out_chans, kernels, exp_chans, strides, use_squeeze, use_swish)
+        ):
             _layers.append(
-                InvertedResidual(out, k, exp, s, use_sq, use_sw),
+                InvertedResidual(out, k, exp, s, use_sq, use_sw, name="inverted_" + str(i)),
```

Comment:
smart move but let's use f-string since we fortunately have python 3.6+:
`name=f"inverted_{i}"`